jsapp/js/components/formLanding.es6

```diff
@@ -177,6 +191,16 @@ export class FormLanding extends React.Component {
             <button className='mdl-button mdl-button--colored' onClick={this.toggleDeploymentHistory}>
               {this.state.historyExpanded ? t('Hide full history') : t('Show full history')}
             </button>
+            {this.state.historyExpanded &&
+              <button className='mdl-button mdl-button--colored' onClick={this.increaseDVCOUNT_LIMIT}>
+                {this.state.DVCOUNT_LIMIT < dvcount && t('Load more')}
+              </button>
+            }
+            {this.state.historyExpanded &&
+               <button className='mdl-button mdl-button--colored' onClick={this.decreaseDVCOUNT_LIMIT}>
+                  {this.state.DVCOUNT_LIMIT - DVCOUNT_LIMIT_MINIMUM >= DVCOUNT_LIMIT_MINIMUM && t('Load less')}
+              </button>
+            }
```

Comment:
Is there really a point in having a "load less" button? I would just keep it simple and have two buttons.